Evaluator/Strategies/mixed_strategies_evaluator/tests/test_simple_mixed_strategies_evaluator.py

```diff
@@ -1,80 +0,0 @@
-#  Drakkar-Software OctoBot-Tentacles
-#  Copyright (c) Drakkar-Software, All rights reserved.
-#
-#  This library is free software; you can redistribute it and/or
-#  modify it under the terms of the GNU Lesser General Public
-#  License as published by the Free Software Foundation; either
-#  version 3.0 of the License, or (at your option) any later version.
-#
-#  This library is distributed in the hope that it will be useful,
-#  but WITHOUT ANY WARRANTY; without even the implied warranty of
-#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-#  Lesser General Public License for more details.
-#
-#  You should have received a copy of the GNU Lesser General Public
-#  License along with this library.
-import pytest
-
-from tests.functional_tests.strategy_evaluators_tests.abstract_strategy_test import AbstractStrategyTest
-from tentacles.Evaluator.Strategies import SimpleStrategyEvaluator
-from tentacles.Trading.Mode import DailyTradingMode
-
-
-# All test coroutines will be treated as marked.
-pytestmark = pytest.mark.asyncio
-
-
-@pytest.fixture()
-def strategy_tester():
-    strategy_tester_instance = _TestSimpleMixedStrategiesEvaluator()
-    strategy_tester_instance.initialize(SimpleStrategyEvaluator, DailyTradingMode)
-    return strategy_tester_instance
-
-
-# TODO rename this test into TestTechnicalAnalysisStrategyEvaluator to include it in pytest tests list when
-# backtesting randomness is fixed (will require tested values fixes)
-class _TestSimpleMixedStrategiesEvaluator(AbstractStrategyTest):
```

Comment:
It's replaced by TestTechnicalAnalysisStrategyEvaluator already ?

Reply:
It's a legacy test that I forgot to remove